tox.ini

```diff
@@ -30,8 +30,11 @@ deps =
 commands =
     {envpython} --version
     trial --version
-    coverage run --source klein --omit klein/test/* --branch {envdir}/bin/trial {posargs:klein}
+    coverage erase
```

Comment:
I intentionally removed all this junk from the `tox.ini` previously because it makes it impossible to collect coverage with `detox`.  Can we keep it out of here?